builder/main.py

```diff
@@ -159,11 +306,27 @@ def generate_uf2(target, source, env):
         UPLOADCMD='"$UPLOADER" $UPLOADERFLAGS $SOURCES'
     )
 
+    if "uploadfs" in COMMAND_LINE_TARGETS:
+        env.Replace(
+            UPLOADER=join(platform.get_package_dir("tool-rp2040tools") or "", "picotool"),
+            UPLOADERFLAGS=[
+                "load",
+                "--verify"
+            ],
+            UPLOADCMD="$UPLOADER $UPLOADERFLAGS $SOURCES --offset ${hex(FS_START)}",
```

Comment:
Does it make sense to wrap `$UPLOADER` by `"` to avoid issues with possible whitespaces in the path to PlatformIO packages?

Reply:
Should the same thing be done with `$SOURCES` to be safe then?

Reply:
I believe `$SOURCES` is handled by SCons internally.

Reply:
Uploader is now quoted.